tests/series-upgrade/tests/bundles/bionic-to-focal-ussuri-ha.yaml.j2

```diff
@@ -0,0 +1,653 @@
+# This is the bionic-ussuri to focal-ussuri series upgrade bundle with
+# percona-cluster (at bionic).
+#
+# This bundle does not intend to upgrade perconal cluster to mysql8
+# (mysql-innodb-cluster and mysql-router) as that will be handled by the
+# perconal-cluster to mysql8 migration test.
+#
+# TODO:
+#         - Add SSL everywhere
+#         - Add HA everywhere
+
+variables:
+  openstack-origin: &openstack-origin cloud:bionic-ussuri
+
+series: bionic
+
+applications:
+
+  aodh:
+    charm: cs:~openstack-charmers-next/aodh
+    num_units: 3
+    options:
+      openstack-origin: *openstack-origin
+      vip: {{ TEST_VIP00 }}
+    constraints: mem=1024
+
+  aodh-hacluster:
+    charm: cs:~openstack-charmers-next/hacluster
+    options:
+      cluster_count: 3
+      corosync_transport: unicast
+
+#  barbican:
```

Comment:
I think we should have a note right here explaining why this is commented out, or just remove that charm. Because I won't be able to tell in the future why this is commented out and what is preventing us from enabling it.

Same in other bundles.